docs/Plugins/tapd.md

```diff
@@ -5,9 +5,19 @@ description: >
 ---
 
 ## Summary
-
 This plugin collects Tapd data through Tapd REST API. 
```

Comment:
Text change:
This plugin collects TAPD data through its REST APIs. TAPD is an issue-tracking tool similar to Jira.